scene/2d/parallax_layer.cpp

```diff
@@ -71,7 +71,7 @@ void ParallaxLayer::_update_mirroring() {
 	if (pb) {
 		RID c = pb->get_canvas();
 		RID ci = get_canvas_item();
-		Point2 mirrorScale = mirroring * get_scale();
+		Point2 mirrorScale = mirroring * orig_scale;
```

Comment:
```suggestion
		Point2 mirror_scale = mirroring * orig_scale;
```
Let's fix the incorrect name while we're at it

Reply:
Lol. Better late than never!